src/main/java/net/atos/client/zgw/shared/model/ObjectType.java

```diff
@@ -5,13 +5,103 @@
+import net.atos.client.zgw.shared.model.audit.zaken.RolMedewerkerWijziging;
+import net.atos.client.zgw.shared.model.audit.zaken.RolNatuurlijkPersoonWijziging;
+import net.atos.client.zgw.shared.model.audit.zaken.RolNietNatuurlijkPersoonWijziging;
+import net.atos.client.zgw.shared.model.audit.zaken.RolOrganisatorischeEenheidWijziging;
+import net.atos.client.zgw.shared.model.audit.zaken.RolVestigingWijziging;
+import net.atos.client.zgw.shared.model.audit.zaken.StatusWijziging;
+import net.atos.client.zgw.shared.model.audit.zaken.ZaakEigenschapWijziging;
+import net.atos.client.zgw.shared.model.audit.zaken.ZaakInformatieobjectWijziging;
+import net.atos.client.zgw.shared.model.audit.zaken.ZaakWijziging;
+import net.atos.client.zgw.shared.model.audit.zaken.ZaakobjectWijziging;
+import net.atos.client.zgw.zrc.model.BetrokkeneType;
+import net.atos.client.zgw.zrc.model.ZaakBesluit;
+
 public enum ObjectType {
-    ZAAK,
-    RESULTAAT,
-    STATUS,
-    ROL,
-    ZAAKOBJECT,
-    ZAAK_INFORMATIEOBJECT,
-    ENKELVOUDIG_INFORMATIEOBJECT,
-    GEBRUIKSRECHTEN
+
+    /** http://open-zaak/zaken/api/v1/zaken/{zaak_uuid}/zaakeigenschappen */
+    ZAAKEIGENSCHAP("/zaakeigenschappen/", ZaakEigenschapWijziging.class),
+
+    /** http://open-zaak/zaken/api/v1/zaken/{zaak_uuid}/besluiten/{uuid} */
+    ZAAKBESLUIT("/besluiten/", ZaakBesluit.class),
+
+    /** http://open-zaak/zaken/api/v1/zaken/{uuid} */
+    ZAAK("/zaken/api/v1/zaken/", ZaakWijziging.class),
+
+    /** http://open-zaak/zaken/api/v1/resultaten/{uuid} */
+    RESULTAAT("/zaken/api/v1/resultaten/", ResultaatWijziging.class),
+
+    /** http://open-zaak/zaken/api/v1/statussen/{uuid} */
+    STATUS("/zaken/api/v1/statussen/", StatusWijziging.class),
+
+    /** http://open-zaak/zaken/api/v1/rollen/{uuid} */
+    ROL("/zaken/api/v1/rollen/", null),
+
+    /** http://open-zaak/zaken/api/v1/zaakobjecten/{uuid} */
+    ZAAKOBJECT("/zaken/api/v1/zaakobjecten", ZaakobjectWijziging.class),
+
+    /** http://open-zaak/zaken/api/v1/zaakinformatieobjecten/{uuid} */
+    ZAAK_INFORMATIEOBJECT("/zaken/api/v1/zaakinformatieobjecten", ZaakInformatieobjectWijziging.class),
+
+    /** http://open-zaak/documenten/api/v1/enkelvoudiginformatieobjecten/{uuid} */
+    ENKELVOUDIG_INFORMATIEOBJECT("/documenten/api/v1/enkelvoudiginformatieobjecten/", EnkelvoudigInformatieobjectWijziging.class),
+
+    /** http://open-zaak/documenten/api/v1/gebruiksrechten/{uuid} */
+    GEBRUIKSRECHTEN("/documenten/api/v1/gebruiksrechten", GebuiksrechtenWijziging.class),
+
+    /** http://open-zaak/documenten/api/v1/objectinformatieobjecten/{uuid} */
+    OBJECT_INFORMATIEOBJECT("documenten/api/v1/objectinformatieobjecten", ObjectInformatieobjectWijziging.class),
+
+    /** http://open-zaak/zaken/api/v1/klantcontacten/{uuid} */
+    KLANTCONTACT("/zaken/api/v1/klantcontacten", KlantContactWijziging.class);
+
+    private final String url;
+
+    public final Type auditClass;
```

Comment:
Ik weet niet of het kan maat volgens mij is de class toch altijd een subclass van AuditWijziging? Kun je dat hier aangeven? Als dat niet kan is ook oke. Was maar een idee.